docs/faq.md

```diff
@@ -271,6 +273,103 @@ There are several ways of doing this:
 - [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) the image built with Jib to have it available in your local Docker daemon.
 - Alternatively, instead of using a Docker daemon, you can run a local container registry, such as [Docker registry](https://docs.docker.com/registry/deploying/) or other repository managers, and point Jib to push to the local registry.
 
+### What would a Dockerfile for a Jib-built image look like?
+
+A Dockerfile that performs a Jib-like build is shown below:
+
+```Dockerfile
+FROM baseImage
+
+COPY libs /
```

Comment:
Hmm, or maybe it'd just be more clear to explicitly reference everything from the project directory. i.e. 

```Dockerfile
COPY build/classes /app/classes
COPY build/libs /app/libs
...
```

Reply:
+1 that works too - maybe even more explicit directory names?

```
COPY classFiles /app/classes
COPY dependencyJars /app/libs
...
```

Since directories like `build/libs` do not automatically contain all the dependency jars, for example.